.eslintrc.js

```diff
@@ -122,59 +107,43 @@ module.exports = {
     "no-implicit-coercion": "error",
     "no-implicit-globals": "error",
     "no-implied-eval": "error",
-    "no-import-assign": "error",
     "no-inline-comments": "error",
-    "no-invalid-this": "off",
     "no-iterator": "error",
     "no-label-var": "error",
     "no-labels": "error",
     "no-lone-blocks": "error",
     "no-lonely-if": "error",
     "no-loop-func": "error",
-    "no-magic-numbers": "error",
+    "no-magic-numbers": ["error", { "ignore": [0, 1] }],
```

Comment:
We should also ignore array indexes.

```suggestion
    "no-magic-numbers": ["error", { "ignore": [0, 1], "ignoreArrayIndexes": true }],
```

Reply:
rebased into https://github.com/AlaskaAirlines/eslint-config-auro/pull/7/commits/8a2b3ce7c50b04cf8e457ff6f5227eb670369eb4